azure-pipelines.yml

```diff
@@ -45,7 +39,7 @@ jobs:
             /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
             /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
             /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-            
```

Comment:
The DotNetSymbolServerTokenMsdl and DotNetSymbolServerTokenSymWeb variables are not needed.  I'm removing them from Arcade in https://github.com/dotnet/arcade/pull/1595/files#diff-fec826feae04e51c0d94076385408bdc